Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -333,6 +333,28 @@ static String mutationObserverNotificationString()
 
     if (m_pdfTestCallback)
         std::exchange(m_pdfTestCallback, nullptr)->handleEvent();
+
+    sizeToFitContents();
```

Comment:
Is it ok to do this after we run the tests or should we do them before?

Reply:
That's a good point. In fact, the test callback should be the last thing to run once we are certain that the document is installed and in a valid state.

Reply:
I'll make that bit a scope exit.

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/e6abaffdcea753bb1711d92bc067f42526af4c96